lms/resources/_js_config/__init__.py

```diff
@@ -365,19 +365,17 @@ def add_deep_linking_api(self):
         self._config["filePicker"]["deepLinkingAPI"] = config
 
     def enable_instructor_dashboard_entry_point(self, assignment):
-        self._config["hypothesisClient"] = self._hypothesis_client
         self._hypothesis_client["dashboard"] = {
             "showEntryPoint": True,
-            "entryPointRPCMethod": "openDashboard",
-        }
-        self._config["dashboard"] = {
-            "dashboardEntryPoint": {
+            "authTokenRPCMethod": "requestAuthToken",
+            "entryPoint": {
                 "path": self._request.route_path(
                     "dashboard.launch.assignment", id_=assignment.id
                 ),
-                "data": {},
             },
+            "authFieldName": "authorization"
```

Comment:
We have discussed that it probably makes sense the field name where the auth token is set is not hardcoded in the FE, but the BE can control it.

However, I'm not sure if this is the best way to define it.